docs/features/okf-context-layer/prd.md

```
@@ -0,0 +1,278 @@
+
+### New Settings Fields
+```typescript
+interface ProjectSettings {
+  // ... existing fields ...
+  domain_keywords?: string[];      // Domain terms AI should use
+  avoided_keywords?: string[];     // Terms AI should avoid
+}
+```
+
+### New API Endpoint
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)